.go-version

```diff
@@ -1 +1 @@
-1.24.2
+1.25.3
```

Comment:
This change should be a separate PR, but included here to allow the tests to actually run.

Reply:
Will be done in https://github.com/elastic/elastic-package/pull/3015.